action-server/tests/test_action_send_daily_checkin_reminder.py

```diff
@@ -98,6 +103,10 @@ def test_happy_path(self, mock_session_factory):
             ),
         )
 
+        self.assertEqual(reminder_mock.last_reminded_at, DEFAULT_NOW)
+        mock_session_factory.return_value.commit.assert_called()
+        mock_session_factory.return_value.close.assert_called()
```

Comment:
There should be the equivalent in unhappy path tests to make sure session.rollback() was called.

Reply:
Right. Done.